docker/pennylane.dockerfile

```diff
@@ -43,7 +43,8 @@ RUN git submodule update --init --recursive \
     && pip install wheel && pip install -r requirements.txt \
     && python3 setup.py install \
     && pip install pytest pytest-cov pytest-mock flaky \
-    && make test
+    && pip install -i https://test.pypi.org/simple/ pennylane-lightning --pre --upgrade \
```

Comment:
Would this line have to be reverted each release cycle than put back in?

Reply:
Since the docker image is built only against the master version of PennyLane, this should always pull in the latest master version of Lightning too.